README.md

```diff
@@ -78,6 +78,14 @@ There are many more parameters you can use. Display the usage doc with the follo
 
     GitLink.exe -h
 
+### Native PDBs
+
+Native PDBs (from C++ projects) are supported by using -a option:
+
+    GitLink.exe <nativePdbfile> -a
+
+All source code files from your git depot will be indexed in the PDB.
```

Comment:
I think it won't have such an impact since it's an explicit thing to enable. (famous last words)

But, if people enabled it *and* find it too slow, then we probably will receive a PR to fix that. If it would be enabled by default, then I would indeed be a bit worried. So maybe this is a good reason that we don't enable this by default in the targets file below and leave as is for now?